airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/OssMetricsRegistry.java

```diff
@@ -87,10 +87,10 @@ public enum OssMetricsRegistry implements MetricsRegistry {
       MetricEmittingApps.METRICS_REPORTER,
       "num_active_conn_per_workspace",
       "number of active connections per workspace"),
-  NUM_ABNORMAL_SCHEDULED_SYNCS(
+  PCT_ABNORMAL_SCHEDULED_SYNCS(
       MetricEmittingApps.METRICS_REPORTER,
       "num_abnormal_scheduled_syncs",
```

Comment:
Can we make the name reflect the time period? e.g. NUM_ABNORMAL_SCHEDULED_SYNCS_LAST_DAY.

We should do the same for the metric name itself.